tests/common_tools.py

```diff
@@ -61,7 +61,9 @@ def create_homepage_in_cms():
                                      name=settings.CONFERENCE_CONFERENCE)
     homepage = create_page(
         title='HOME', template='django_cms/p5_homepage.html', language='en',
-        reverse_id='home', published=True, publication_date=timezone.now())
+        reverse_id='home', published=True, publication_date=timezone.now(),
```

Comment:
since the list of the arguments grew a lot, maybe we could change to one-arg-per line here?